app/src/main/java/com/example/lhi06/residue/MainActivity.java

```diff
@@ -0,0 +1,54 @@
+package com.example.lhi06.residue;
+
+import android.os.Bundle;
+import android.support.design.widget.FloatingActionButton;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+public class MainActivity extends AppCompatActivity {
+
+    @BindView(R.id.add_review) FloatingActionButton addReviewButton;
+    @BindView(R.id.request) EditText searchField;
+    @BindView(R.id.search_button) Button searchButton;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+        ButterKnife.bind(this);
+
+        searchButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                //TODO: Code for searching for reviews done
+                //String artistName = editText1.getText().toString();
+                //handler.urlBuilder(artistName);
+            }
+        });
+
+        addReviewButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                //TODO: Navigate to the album searching activity
+
+            }
+        });
+
+    }
+
+    @Override
+    public boolean onCreateOptionsMenu(Menu menu) {
```

Comment:
You got rid of the toolbar so you don't have options menu anymore so you can get rid of this method and `menu_main.xml`